src/utils/staking/param.ts

```diff
@@ -0,0 +1,39 @@
+import { StakingParams, VersionedStakingParams } from "../../types/params";
+
+/*
+  Get the BBN param by BTC height
+  @param height - The BTC height
+  @param bbnParams - The BBN params
```

Comment:
```suggestion
  @param babyParamsVersions - The Babylon params versions
```

We do not have "params" with singular "param" contents. We have params versions, with each version containing plural params.